src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/ExpressionBinder.cs

```diff
@@ -1275,19 +1275,19 @@ private void AdjustCallArgumentsForParams(CType callingObjectType, CType type, M
             }
         }
 
-        private static readonly PredefinedType[] s_rgptIntOp =
-        {
+        private static ReadOnlySpan<PredefinedType> RgptIntOp() =>
```

Comment:
Any particular reason this is a method and not a property?

Reply:
It could be a property.

Reply:
Just asking because of consistency